openwebstart/src/main/java/com/openwebstart/jvm/LocalRuntimeManager.java

```diff
@@ -135,6 +143,18 @@ void loadRuntimes() {
         }
     }
 
+    private boolean isUnused(final LocalJavaRuntime localJavaRuntime) {
+        final int maxDaysUnusedInJvmCache = RuntimeManagerConfig.getMaxDaysUnusedInJvmCache();
+        final LocalDateTime lastUsage = localJavaRuntime.getLastUsage();
+            final LocalDateTime today = LocalDateTime.now();
+            if (lastUsage.plusDays(maxDaysUnusedInJvmCache).isBefore(today)) {
+                LOG.info(String.format("Delete unused runtime '%s' from JVM cache as it exceeds max number of %s days " +
```

Comment:
Indentation

Also, LOG.info() supports replacing `{}` placeholders. so no need to use String.format

Reply:
fixed